samples/README.md

```diff
@@ -4,53 +4,55 @@ This sample demonstrates how to use Gluon Maps inside a JavaFX application.
 
 ## Instructions
 
-We use [GluonFX plugin](https://docs.gluonhq.com/) to build a native image for platforms including desktop, android, iOS and embedded.
-Please follow the GluonFX prerequisites as stated [here](https://docs.gluonhq.com/#_requirements).
+To run the application, simply execute the following command from the `samples` directory:
 
-### Desktop
+    mvn javafx:run
 
-Run the application using:
+## Native Image
 
-    ./gradlew run
+[GluonFX plugin](https://docs.gluonhq.com/) is used to build a native image for platforms including desktop, android, iOS and embedded.
+Please follow the GluonFX prerequisites as stated [here](https://docs.gluonhq.com/#_requirements).
+
+### Desktop
 
 Build a native image using:
 
-    ./gradlew nativeBuild
+    mvn gluonfx:build
 
 Run the native image app:
 
-    ./gradlew nativeRun
+    mvn gluonfx:nativerun
 
 ### Android
 
 Build a native image for Android using:
 
-    ./gradlew nativeBuild -Ptarget=android
+    mvn gluonfx:build -Ptarget=android
```

Comment:
`-Ptarget=android` was using a gradle property, but with maven you either add a profile on the sample's pom (and then `-Pandroid`, or you can use `-Dgluonfx.target=android` instead.